README.md

```diff
@@ -369,9 +369,9 @@ This is what you should expect from Netdata:
 -   **Zero disk I/O** - while it runs, it does not load or save anything (except `error` and `access` logs).
 -   **Zero configuration** - auto-detects everything, it can collect up to 10000 metrics per server out of the box.
 -   **Zero maintenance** - You just run it, it does the rest.
--   **Zero dependencies** - it is even its own web server, for its static web files and its web API (though its plugins may require additional libraries, depending on the applications monitored).
+-   **Zero dependencies** - it is even its web server, for its static web files and its web API (though its plugins may require additional libraries, depending on the applications monitored).
```

Comment:
`it is even its web server` isn't very clear

Reply:
Perhaps this should read something more like: 

```suggestion
-   **Zero dependencies** - Netdata runs a custom web server for its static web files and its web API (though its plugins may require additional libraries, depending on the applications monitored).
```